sql/src/main/antlr/OpenSearchSQLParser.g4

```diff
@@ -244,6 +244,7 @@ datetimeConstantLiteral
     : CURRENT_DATE
     | CURRENT_TIME
     | CURRENT_TIMESTAMP
+    | DAY_OF_WEEK
```

Comment:
I don't understand why we don't also need `DAYOFWEEK` defined here.

Reply:
Fixed in https://github.com/Bit-Quill/opensearch-project-sql/pull/190/commits/e571bf04368910cfc8be87e69e2cf901bad7a46e